Makefile

```diff
@@ -71,27 +71,27 @@ export OAUTH2_PROXY_IMG ?= $(OAUTH2_PROXY_NAMESPACE)/$(OAUTH2_PROXY_NAME):$(OAUT
 OPENSHIFT_OAUTH_PROXY_NAMESPACE ?= quay.io/cryostat
 OPENSHIFT_OAUTH_PROXY_NAME ?= openshift-oauth-proxy
 # there is no 'latest' tag for this container
-OPENSHIFT_OAUTH_PROXY_VERSION ?= 3.0.0
+OPENSHIFT_OAUTH_PROXY_VERSION ?= cryostat-v3.0
 export OPENSHIFT_OAUTH_PROXY_IMG ?= $(OPENSHIFT_OAUTH_PROXY_NAMESPACE)/$(OPENSHIFT_OAUTH_PROXY_NAME):$(OPENSHIFT_OAUTH_PROXY_VERSION)
 DATASOURCE_NAMESPACE ?= $(DEFAULT_NAMESPACE)
 DATASOURCE_NAME ?= jfr-datasource
-DATASOURCE_VERSION ?= 3.0.0
+DATASOURCE_VERSION ?= 3.0.1-snapshot
 export DATASOURCE_IMG ?= $(DATASOURCE_NAMESPACE)/$(DATASOURCE_NAME):$(DATASOURCE_VERSION)
 GRAFANA_NAMESPACE ?= $(DEFAULT_NAMESPACE)
 GRAFANA_NAME ?= cryostat-grafana-dashboard
-GRAFANA_VERSION ?= 3.0.0
+GRAFANA_VERSION ?= cryostat-v3.0
```

Comment:
I don't see this style of tag for Grafana anymore. I think we used ` 3.0-dev` for Helm

Reply:
whoops, good catch

Reply:
I think this needs a `make generate bundle` as well